mica/stats/update_guide_stats.py

```diff
@@ -112,7 +115,7 @@
         ('var', 'int'),
         ('pos_err', 'int'),
         ('mag_aca', 'float'),
-        ('mag_err', 'int'),
+        ('mag_aca_err', 'int'),
```

Comment:
Hi @taldcroft  and @javierggt ! It sounds like you also want a change in this PR somewhere to scale the mag_aca_err column to units of mag.  That's fine.  Though should that be in mica or in agasc.get_star (or get_stars) with a calibrate flag or some such?

Reply:
Well given that all the rest of this is uncalibrated it appears we cannot make a change here without having a one-off inconsistency.  Just leave it.  And don't bother with making an `agasc` issue.  If one of us gets annoyed enough one day we'll do it, but it is a low priority and we don't need more low priority issues.

Reply:
Would it be less inconsistent to just UPPER CASE these?

Reply:
It would be more consistent to upper case them, but would break all existing code that uses these.  Not worth the trouble I'd say.